src/USER-PLUMED/fix_plumed.h

```diff
@@ -0,0 +1,59 @@
+
+#ifdef FIX_CLASS
+
+FixStyle(plumed,FixPlumed)
+
+#else
+
+#ifndef LMP_FIX_PLUMED_H
+#define LMP_FIX_PLUMED_H
+
+#include "fix.h"
+#include "compute.h"
+// the plumed header that defines the class//
+#include "Plumed.h"
```

Comment:
Please do not include any specific headers at this point outside of what is absolutely needed. For pointers to objects usually a simple forward declaration is sufficient and will help us to keep the namespace clean.